.vscode/settings.json

```diff
@@ -10,5 +10,121 @@
         "data/schema/recipes.json": [
             "data/recipes.yaml"
         ],
-    }
+    },
+    "cSpell.words": [
+        "accum",
+        "antiquotation",
+        "antiquoted",
+        "antiquoting",
+        "assocs",
+        "autoplay",
+        "Bifunctor",
+        "bimap",
+        "Blackhole",
+        "Blackholes",
+        "bquote",
+        "Bytestring",
+        "callsite",
+        "cata",
+        "CESK",
+        "Cmds",
+        "Colour",
+        "Combinators",
+        "conv",
+        "Corasick",
+        "datatypes",
+        "determinator",
+        "determinators",
+        "distrib",
+        "eithers",
+        "elems",
+        "ents",
+        "floorplan",
+        "Focusable",
+        "foldl",
+        "foldr",
+        "Gameplay",
+        "Hashable",
+        "hifi",
+        "HLINT",
+        "homomorphic",
+        "hsep",
+        "Ixed",
+        "JSONE",
+        "Keymap",
+        "kolor",
+        "leaderboard",
+        "mappend",
+        "mconcat",
+        "mempty",
+        "Monoid",
+        "multiset",
+        "multiworld",
+        "objs",
+        "omni",
+        "Parameterizable",
+        "parens",
+        "pathfinding",
+        "Perlin",
+        "perp",
+        "playfield",
+        "Polytype",
+        "polytypes",
+        "pparens",
+        "prec",
+        "precomputation",
+        "Precompute",
+        "preprocess",
+        "prereq",
+        "prereqs",
+        "Prettyprinter",
+        "preuse",
+        "previewable",
+        "pushable",
+        "quasiquoter",
+        "quasiquoters",
+        "sconcat",
+        "scrollability",
+        "selfdestruct",
+        "Semigroup",
+        "snoc",
+        "Splittable",
+        "squote",
+        "SRGB",
+        "struc",
```

Comment:
Is this entry supposed to be here?  What is `struc`?

Reply:
It's debatable.  It's my truncated version of "structure".  The fact that you asked suggests that I should rename it in the source code :)